src/main/java/com/maxmind/minfraud/response/Email.java

```diff
@@ -0,0 +1,50 @@
+package com.maxmind.minfraud.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This class contains minFraud response data related to the email address.
+ */
+public final class Email {
+    @JsonProperty("is_free")
```

Comment:
You don't need these. Jackson can't even access the attributes directly as we turned off overriding of access modifiers.